README.md

```diff
@@ -22,7 +22,7 @@ Download and install **Android WiFi ADB** directly from Intellij / Android Studi
 Alternatively, you can [download the plugin][6] from the JetBrains plugin site and install it manually in:
 `Preferences/Settings->Plugins->Install plugin from disk`.
 
-**The plugin is waiting for approval by the IntelliJ Plugin Store developers** If you don't find the plugin the IntelliJ plugin store you can [download here][6].
+**The plugin is waiting for approval by the IntelliJ Plugin Store developers** If you don't find the plugin the IntelliJ plugin store you can [download here][7].
```

Comment:
@sebastianmarschall can you remove this line and also the link  number [7]. Once the plugin has been published we don't need this link anymore. Thanks!